javascript/src/seedsource-ui/containers/RegionStep.tsx

```diff
@@ -25,6 +25,20 @@ type RegionStepProps = ConnectedProps<typeof connector> & {
   number: number
 }
 
+const automaticBold = <strong>{t`Automatic`}</strong>
+const customBold = <strong>{t`Bold`}</strong>
+
+const helpTooltip = (
+  <p
+    dangerouslySetInnerHTML={{
+      __html: t`The map is broken into large geographic regions such as the Western US and the Eastern US.
+    Select <strong>Automatic</strong> if your seedlot(s) and planting site(s) are from
+    the same region (most common). Select <strong>Custom</strong> if they are from different regions,
+    and then specify the region that does not include the location chosen in step 2.` as string,
+    }}
+  />
+)
```

Comment:
You don't need to set the inner HTML, though you may need to use `jt` instead of `t`. Take a look at `Menu.tsx` for examples.